software/mk/Makefile.builddefs

```diff
@@ -174,7 +174,6 @@ RISCV_LINK = $(RISCV_GCC) -t -T $(LINK_SCRIPT) $(RISCV_LINK_SYMS)
 
 
 ifndef CLANG
-  RISCV_GCC_OPTS += -mno-fdiv
   RISCV_GCC_OPTS += -fweb -frename-registers -frerun-cse-after-loops
```

Comment:
Shouldn't this be `-frerun-cse-after-loop` ?

Reply:
I have a separate PR that fixes that.